ick/config/rules.py

```diff
@@ -96,6 +96,9 @@ class RuleConfig(Struct):
     outputs: Optional[Sequence[str]] = None
     extra_inputs: Optional[Sequence[str]] = None
 
+    description: Optional[str] = None
+    contact: Optional[str] = None
```

Comment:
We'll probably also want a default section that sets the contact for the whole toml file, but that can come later.